charts/elasticsearch-index/templates/job.yaml

```diff
@@ -31,16 +31,13 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: curl
-          image: appropriate/curl
-          command:
-            - /bin/sh
-            - -c
-            - source /tmp/templates/init.sh
-          volumeMounts:
-            - mountPath: /tmp/templates
-              name: es-init-volume
-
-      volumes:
-        - name: es-init-volume
-          configMap:
-            name: es-init-config
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          args:
+            - create
```

Comment:
Can you add a comment here that `create` actually means `create-if-not-exists`? It's not obvious from reading these arguments.